pkg/plugins/runtime/k8s/metadata/annotations_test.go

```diff
@@ -9,11 +9,13 @@ import (
 
 var _ = Describe("Kubernetes Annotations", func() {
 
-	Context("GetEnabled()", func() {
+	Describe("GetEnabled()", func() {
```

Comment:
Can we rewrite this test to use `DescribeTable`?

Reply:
yes, i thought the same. will rewrite the test 👍

Reply:
done